tools/libvirt_test.sh

```diff
@@ -82,6 +82,15 @@ BLUEPRINT_FILE=${TEMPDIR}/blueprint.toml
 COMPOSE_START=${TEMPDIR}/compose-start-${IMAGE_KEY}.json
 COMPOSE_INFO=${TEMPDIR}/compose-info-${IMAGE_KEY}.json
 
+# Create SSH key
+SSH_KEY=/tmp/id_rsa
+ssh-keygen -f "${SSH_KEY}" -N "" -q
+
+# Change cloud-init/user-data ssh key
+key=" -ssh-rsa $(sed "1d;38d;s/\n//g" "${SSH_KEY}"| tr -d '[:space:]')"
+sed -i "/ssh-rsa/d" /usr/share/tests/osbuild-composer/cloud-init/user-data # In case there's a key for some reason
```

Comment:
You should remove it in the first place in the original file `test/data/cloud-intit/user-data` so you don't have to worry about this.

Reply:
Writing into `/usr` always feels weird, can we do this in a temporary directory?